client/relay-chain-rpc-interface/src/reconnecting_ws_client.rs

```diff
@@ -64,6 +64,18 @@ pub struct ReconnectingWsClient {
 	to_worker_channel: TokioSender<RpcDispatcherMessage>,
 }
 
+/// Format url and force addition of a port
+fn url_to_string_with_port(url: Url) -> Option<String> {
+	Some(format!(
+		"{}://{}:{}{}{}",
+		url.scheme(),
+		url.host_str()?,
+		url.port_or_known_default()?,
+		url.path(),
+		url.query().map(|query| format!("?{}", query)).unwrap_or_default()
```

Comment:
I am not sure if query parameters are allowed on websocket URLs (if so sounds super niche), quick google did not help much :thinking:.

Reply:
Are allowed.  Section 3 of https://www.rfc-editor.org/rfc/rfc6455